tailwind.config.js

```diff
@@ -134,6 +134,9 @@ module.exports = {
             fcsGreen: '#345d34',
             fcsOrange: '#ea6a2c',
             fcsRed: '#fa190e',
+            ipcStartColor: '#F6D1C1',
+            ipcMiddleColor: '#FB7453',
+            ipcEndColor: '#710013',
```

Comment:
`ipsStart`, `ipsMiddle` and `ipsEnd` should be enough - none of the colors in this file have the suffix `color`.